e2e/process_attachmentList_actionMenu.e2e.ts

```diff
@@ -0,0 +1,205 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import LoginPage = require('./pages/adf/loginPage');
+import ProcessServicesPage = require('./pages/adf/process_services/processServicesPage');
+import ProcessFiltersPage = require('./pages/adf/process_services/processFiltersPage.js');
+import ProcessDetailsPage = require('./pages/adf/process_services/processDetailsPage.js');
+import AttachmentListPage = require('./pages/adf/process_services/attachmentListPage.js');
+import ViewerPage = require('./pages/adf/viewerPage.js');
+
+import CONSTANTS = require('./util/constants');
+
+import TestConfig = require('./test.config');
+import resources = require('./util/resources');
+import Util = require('./util/util.js');
+
+import path = require('path');
+
+import AlfrescoApi = require('alfresco-js-api-node');
+import { UsersActions } from './actions/users.actions';
+import { AppsActions } from './actions/APS/apps.actions.ts';
+import FileModel = require('./models/ACS/fileModel.js');
+
+describe('Attachment list action menu for processes', () => {
+
+    let loginPage = new LoginPage();
+    let processServicesPage = new ProcessServicesPage();
+    let processFiltersPage = new ProcessFiltersPage();
+    let processDetailsPage = new ProcessDetailsPage();
+    let attachmentListPage = new AttachmentListPage();
+    let viewerPage = new ViewerPage();
+    let app = resources.Files.SIMPLE_APP_WITH_USER_FORM;
+    let jpgFile = new FileModel({
+        location: resources.Files.ADF_DOCUMENTS.JPG.file_location,
+        name: resources.Files.ADF_DOCUMENTS.JPG.file_name
+    });
+    let pdfFile = new FileModel({
+        location: resources.Files.ADF_DOCUMENTS.PDF.file_location,
+        name: resources.Files.ADF_DOCUMENTS.PDF.file_name
+    });
+
+    let downloadedJpgFile = path.join(__dirname, '\\downloads', jpgFile.name);
```

Comment:
let downloadedJpgFile = path.join(__dirname, '/downloads', jpgFile.name);
can you change it like that?

Reply:
path.join in node.js takes care about all slashes, it should be
```
path.join(__dirname, 'downloads', jpgFile.name)
```